src/django/pfb_analysis/models.py

```diff
@@ -197,6 +197,7 @@ class Status(object):
                                                 'e.g. http://a.com/foo.osm or ' +
                                                 'http://a.com/foo.osm.bz2')
     overall_scores = JSONField(db_index=True, default=dict)
+    census_block_count = models.IntegerField(blank=True, null=True)
```

Comment:
PositiveIntegerField is likely preferable here, but shouldn't be a big deal either way.

https://docs.djangoproject.com/en/1.10/ref/models/fields/#positiveintegerfield